themes/rainbowstone/README.md

```diff
@@ -4,10 +4,10 @@
 # Rainbowstone theme
 This back-end OFBiz theme is bootstrap based and offers 4 colour variants. As an example
 
-<img src="http://svn.apache.org/repos/asf/ofbiz/ofbiz-framework/trunk/themes/rainbowstone/webapp/rainbowstone/images/themeRuby.png" alt="Rainbowstone ruby colour variant" />
+<img src="https://raw.githubusercontent.com/apache/ofbiz-framework/trunk/themes/rainbowstone/webapp/rainbowstone/images/themeRuby.png" alt="Rainbowstone ruby colour variant" />
 
 ## License
-The Rainbowstone theme uses the Quicksand font which is licensed under the [SIL OPEN FONT LICENSE Version 1.1](https://svn.apache.org/repos/asf/ofbiz/ofbiz-framework/trunk/themes/rainbowstone/webapp/rainbowstone/fonts/quicksand/Quicksand SIL OFL License 1.1.txt "SIL OPEN FONT LICENSE Version 1.1")
+The Rainbowstone theme uses the Quicksand font which is licensed under the [SIL OPEN FONT LICENSE Version 1.1](https://raw.githubusercontent.com/apache/ofbiz-framework/trunk/LICENSE)
```

Comment:
This line may need to be updated, as mention of SIL OFL in LICENSE is removed recently in commit 180b24.

Reply:
Thanks Wang,

I made some changes at 
https://github.com/apache/ofbiz-framework/commit/a80c12ae3344c2c899564e5f75dfd975240c9115/ 
+
https://github.com/apache/ofbiz-framework/commit/2cf1e9d490234e7758c0d1b937d50eaa0367173d/